<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMain</class>
 <widget class="QMainWindow" name="FormMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
    <item>
     <widget class="QCustomPlot" name="qcp_plot" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Gen method</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_genMethod">
        <item>
         <property name="text">
          <string>UniformIntDistributionUniq LinearDoobleGen</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>UniformIntDistributionUniq NonLinearEqualChanceRange</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>linear</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>linear x3</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_x_axis_scale">
        <property name="text">
         <string>0..1</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,1,0">
      <item>
       <widget class="QLabel" name="l_repeatCount">
        <property name="text">
         <string>RepeatCount</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sp_repeatCount">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000000000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
        <property name="value">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Bars count</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sp_barsCount">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_gen">
        <property name="text">
         <string>Gen</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
